test/TensorFlow/graph_op_inst.sil

```diff
@@ -0,0 +1,69 @@
+// RUN: %target-sil-opt -sil-print-all %s | %FileCheck %s
```

Comment:
also add some negative test cases?
e.g. P.diagnose(P.Tok, diag::expected_tok_in_sil_instr, "graph_op name");

Reply:
There isn't a tradition of testing SIL parser errors, unfortunately. I'm not able to write an effective unit test because introducing one error causes many unrelated parser errors to occur.
SIL verification handles errors instead.

Reply:
Ok. Agree that's unfortunate.